configure.ac

```diff
@@ -319,6 +319,18 @@ VERSION_ARG(VERSION_NUMBER)
 PACKAGE_VERSION=$PACEMAKER_VERSION
 VERSION=$PACEMAKER_VERSION
 
+# Detect highest API schema version (use git if available to list managed RNGs,
+# in case there are leftover schema files from an earlier build of a different
+# version, otherwise check all RNGs)
+API_VERSION=$({ git ls-files xml/api/*.rng 2>/dev/null || ls -1 xml/api/*.rng ; } dnl
```

Comment:
Nice!

It'd be nice to see something like this in the schema generation code too at some point, if we're not already doing it. I occasionally fight weird issues and then realize it built the schemas based on a leftover newer version.